Update Contribution Guide (#818)
grammyjs/website: CONTRIBUTING.md

4. All translation happens and the labels for the respective languages are added.
5. The PR now has at least one approval and all translations are ready, so it can be merged.

> > Make sure that your PRs have the “Allow edits by maintainers” option checked, so that we are able finalize and accept your PR in case of your disappearance.

Reviewer: Let's not make “we don't trust you to stick around until we merge” one of the first statements we say to contributors.

```suggestion
> Make sure that your PRs have the “Allow edits by maintainers” option checked, so that we are able finalize and accept your pull request.
```
